CHANGELOG.md

```diff
@@ -42,6 +43,40 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 2.7.0
+
+### Change
+
+- change: check metadata_schema with check_schema like the other schema [#4381](https://github.com/apache/apisix/pull/4381)
+- change(echo): remove odd auth_value [#4055](https://github.com/apache/apisix/pull/4055)
+- fix(admin): correct the resources' count field and change its type to integer [#4385](https://github.com/apache/apisix/pull/4385)
```

Comment:
This should be put under the `### Fix` ?

Reply:
This fix is a break change so I think it should be more noticeable.

Reply:
OK, gotcha.